log-sink/src/main/java/org/springframework/cloud/stream/module/log/LogSinkProperties.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.log;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Gary Russell
+ *
+ */
+@ConfigurationProperties
+public class LogSinkProperties {
+
+	private String name = "log.sink"; // TODO: set from environment 'group' if available
```

Comment:
in other cases, we've used `$application.name` (with a default like this value) for now, e.g.:

https://github.com/spring-cloud/spring-cloud-stream-modules/blob/master/hdfs-sink/src/main/java/org/springframework/cloud/stream/module/hdfs/sink/HdfsSinkProperties.java#L47